lambdas/accesstoken/src/main/java/uk/gov/di/ipv/cri/address/api/handler/AccessTokenHandler.java

```diff
@@ -0,0 +1,117 @@
+package uk.gov.di.ipv.cri.address.api.handler;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import com.nimbusds.common.contenttype.ContentType;
+import com.nimbusds.oauth2.sdk.AccessTokenResponse;
+import com.nimbusds.oauth2.sdk.AuthorizationCodeGrant;
+import com.nimbusds.oauth2.sdk.ErrorObject;
+import com.nimbusds.oauth2.sdk.OAuth2Error;
+import com.nimbusds.oauth2.sdk.ParseException;
+import com.nimbusds.oauth2.sdk.TokenRequest;
+import com.nimbusds.oauth2.sdk.TokenResponse;
+import com.nimbusds.oauth2.sdk.http.HTTPRequest;
+import org.apache.http.HttpStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
Are we using loggers from [apache](https://github.com/alphagov/di-ipv-cri-address-api/blob/4969c03c3c29ea9300fbce9a20ebbd5c5f28a6e9/lambdas/session/src/main/java/uk/gov/di/ipv/cri/address/api/handler/SessionHandler.java#L9) framework or the `slf4j`